### ISSUE TITLE
Add an on screen dialogue system

### ISSUE THREAD

**Issue by erbridge** [to-the-end/to-the-end#80]
I imagine this being a speech bubble (or similar) attached to the player in the world, and text along one or more of the edges for the antagonist's speech.

**Comment by duncjam** [comment]
Yeah I think speech bubbles or old-school RPG text boxes make sense - perhaps they could be colour coded, once we have a sprite for the girl?

Yeah i think that makes sense in terms of the antagonist - how do we indicate that the antagonist is the environment? Room shake while talking!?

**Comment by erbridge** [comment]
I was thinking that we actually replace the antagonist player's mouse pointer with a ball of light or something (if we keep the game two player), but I like the room shaking idea.

**Comment by erbridge** [comment]
Improvements:

- [ ] precalculate the text size to prevent the text from shifting
- [ ] align the text better (and put it in speech bubbles?)
- [ ] prevent text being clipped by the screen

Other ideas?
